src/layouts/SiteDashboard/SiteDashboard.stories.tsx

```diff
@@ -118,4 +123,58 @@ RequestApproved.parameters = {
   },
 }
 
+export const SiteLaunchStepsCompleted = Template.bind({})
+SiteLaunchStepsCompleted.decorators = [
+  (Story) => {
+    return (
+      <MemoryRouter initialEntries={["/sites/storybook/dashboard"]}>
+        <Route path="/sites/:siteName/dashboard">
```

Comment:
nit: for the stories here, we can actually omit the `SiteLaunchProvider` defined up above and define them individually. also, idt we need the extra `MemoryRouter` + `Router` on the new stories cos it seems like they are the same

Reply:
arh right, removed the redundant codes below as well!